README.md

```diff
@@ -11,7 +11,7 @@ This repository is the official home of the Move virtual machine, bytecode verif
 
 ## Community
 
-* Join us on the [Move Discord](https://discord.gg/M95qX3KnG8).
+* Join us on the [Move Discord](https://discord.gg/vTepKPdF).
```

Comment:
Is that the public or contributor-only channel? (If such a thing exists?)

Reply:
This will invite them to the server. The server contains channels for both.